README.md

```diff
@@ -183,7 +183,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [X](#x) | [Y](#y) | [Z](#z) |
 
 - ### **A**
-
+  - [Belal Ahmad](https://github.com/belaletech)
```

Comment:
Please add your name alphabetically and in the B section.
Let us know if you need any help with that. 🙂